doc/getting_started/index.rst

```diff
@@ -157,7 +157,8 @@ Set Up a Toolchain
    In some specific configurations like non-MCU x86 targets on Linux,
    you may be able to re-use the native development tools provided by
    your operating system instead of an SDK by setting
-   ``ZEPHYR_TOOLCHAIN_VARIANT=host``.
+   ``ZEPHYR_TOOLCHAIN_VARIANT=host`` for gcc or
+   ``ZEPHYR_TOOLCHAIN_VARIANT=llvm`` for clang.
```

Comment:
Hmmm.... I think you're getting some file changes pulled in that aren't part of your PR.  Rebase needed?

Reply:
@dbkinder @jukkar 

I would like to fuzz combinations of a few fields in TCP_ESTABLISHED.

One way is to use libFuzzer from clang with the echo application compiled as a native_posix target (libFuzzer requires clang specific instrumentation and llinking with the libfuzzer library).

This commit is temporary and will be dropped in case it’s not feasible or the present cmake logic will be sufficient to achieve this.

Just ignore it for the time being.